modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/higher_level_reviews_controller.rb

```diff
@@ -37,6 +38,9 @@ def schema
   end
 
   def show
+    if status_simulation_reqested? && status_simulation_allowed?
+      @higher_level_review = status_simulation_for(@higher_level_review)
```

Comment:
I like this pattern, but my sense is that the method name `status_simulation_for` doesn't indicate that the return value is a wrapped object. What do you think?

Reply:
Yeah, that is a good point... Do you have an idea for a more fitting name?